dev-tools/Dockerfile

```diff
@@ -0,0 +1,25 @@
+
+FROM ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder:latest
+
+RUN apk add python3 \
+            py3-qt5 \
+            tesseract-ocr \
+            qemu-arm \
+            libusb
+
+ARG QEMU_BIN=qemu-arm
+
+# -static version is not available in Alpine repositories
+RUN ln -s "$QEMU_BIN" $(dirname $(which "$QEMU_BIN"))/$QEMU_BIN-static
+
+# Python packages building dependencies, can be removed afterwards
+ARG PYTHON_BUILD_DEPS=python3-dev,musl-dev
```

Comment:
Please order all the packages in this list by alphabetical order as well.

Reply:
Done.